src/SimCan.cpp

```diff
@@ -0,0 +1,26 @@
+#include "SimCan.h"
+
+SimCan::SimCan() {
+    _can = new mcp2515_can(CAN_CS_PIN);
+}
```

Comment:
It might be more extensible to pass an instance of the CAN object here in case we want to create different CAN simulators.

Reply:
That's still to do, but will happen in another PR.

Reply:
Sounds good!